book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] it would be better if you add the `add_rental` method to add a book to a rental  in order to achieve `has_many` rentals

Reply:
Noted and implemented as required